test/unit/parser_test.rb

```diff
@@ -98,4 +98,12 @@ def test_arg1_for_push_command
     parser.advance
     assert_equal("constant", parser.arg1)
   end
+
+  def test_arg2_for_push_command
+    input_file = StringIO.new("push constant 17")
+    parser = Parser.new(input_file)
+
+    parser.advance
+    assert_equal(17, parser.arg2)
```

Comment:
You could (if you want to) add `#arg2` assertions to some of the earlier tests that are incidentally checking the command contents as part of tests for `#has_more_commands?` etc.

Reply:
Thankfully the same command was used... 😂 
```
    assert_equal(:C_PUSH, parser.command_type)
    assert_equal("constant", parser.arg1)
    assert_equal(17, parser.arg2)
```